mscore/plugin/api/elements.h

```diff
@@ -234,7 +234,7 @@ class Element : public Ms::PluginAPI::ScoreElement {
       API_PROPERTY( glissandoStyle,          GLISSANDO_STYLE           )
       API_PROPERTY( fretStrings,             FRET_STRINGS              )
       API_PROPERTY( fretFrets,               FRET_FRETS                )
-      API_PROPERTY( fretBarre,               FRET_BARRE                )
+      /*API_PROPERTY( fretBarre,               FRET_BARRE                )*/
```

Comment:
Why not `API_PROPERTY( fretNut,               FRET_NUT               )`?

Reply:
Good point, I missed this line. I'll fix it.